agir/notifications/components/push/subscriptions.js

```diff
@@ -121,8 +121,8 @@ const useWebPush = () => {
     unsubscribe,
   } = useServerSubscription(
     SUBSCRIPTION_TYPES.WEBPUSH,
-    browserSubscription.registration_id,
-    ...browserSubscription
+    browserSubscription?.registration_id,
+    ...(browserSubscription || {})
```

Comment:
Franchement il faudrait proposer d'ajouter un nouvel opérateur : 
```suggestion
    ???browserSubscription
```